### PR TITLE
Created clone for mint.com signup page

### PR DESCRIPTION
**Pull request by anewman15**
We created a clone for mint.com signup page. We added the logos, a signup form area, terms and conditions declaration and copyright declaration sections.

### CODE REVIEW

**Comment by Dandush03** [comment]
# :red_circle: Status: Status Required :disappointed: :red_circle:
### Hey @anewman15 and @beyk    :wave: 
## Good attempts on this one :muscle: :+1: 

## Your project looks great! but is a very important thing you have to consider before submitting another code review :point_down: 

- [ ]  :one: You need to [Set-up Linter](https://github.com/microverseinc/linters-config/tree/master/css) in order to get this project approved.
  > I can see that you had added a stickler file but your PR is not running through the checks, I recommend you guys to check the [stickler](https://stickler-ci.com/) page to see if you had activated your project in there.


## Kindly address all these changes and submit a new code review

### Happy coding:computer: 

#### Daniel Laloush

**Comment by anewman15** [comment]
Hi @Dandush03,

Thanks for your feedback. We activated our repo from the Stickler page. We have one PR open but it says we have none open. I can see the Checks tab with 0 issues, but the checks info seems to be loading. Is everything ok with it?

Thanks once again.

Abdullah

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+# Html-Forms
+In this project, we cloned the Sign Up page for mint.com. We created the logos sections, form area and footer section.
+
+In this project, we created:
+
+1. Sign in section at the top-right part of the page.
+2. Sign up form as the main section of the page. We have three input fields and a create account button. We also have privacy policy and terms of service declaration in a paragraph inside the form area.
+3. Footer section with copyright declaration at the bottom of the page.
+
+![App screenshot](images/mint-screenshot.png)
```
Comment (christianotieno) [inline README.md]: # [OPTIONAL] - But I strongly recommend.
Kindly in the screenshot IMG add a dot at the beginning of the URL of the image to be accessible to all. I cant see it from my end when I clone your repo. 

replace this `App screenshot` with this  `App screenshot`



This is what I mean: 👇 

SCREENSHOT

File: mint-signup-form.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                <a href="#">Privacy</a>
+                <a href="#">Security</a>
+            </div>
+            <div id="copy-right">
+                <p>© 2020 Intuit, Inc. All rights reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.
+                Terms and conditions, features, support, pricing, and service options subject to change without notice.
+                </p>
+            </div>
+            <p>
+                <a href="http://jigsaw.w3.org/css-validator/check/referer">
```
Comment (christianotieno) [inline mint-signup-form.html]: # [REQUIRED] - REMOVE THE VALIDATOR TAG
 This section is not required, Kindly remove it.
```suggestion
                <a href="http://jigsaw.w3.org/css-validator/check/referer">
```
---
This is what it is doing to your page: 👇 
 
w3c

File: mint-signup-form.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            </div>
+            <div>
+                <button id="sign-in-btn" class="btn" type="submit"  value="Signin"><i class="fa fa-lock"></i> Sign in</button>
+                    
+            </div>            
+        </header>
+        
+        <main>
+            <div id="logos">
+                <div id="site-logo">
```
Comment (christianotieno) [inline mint-signup-form.html:24]: # [REQUIRED] - LOGO SECTION
## The logo seems not to be in its right place.
- It should be rightly placed in the top right corner. Kindly see to that. It is a requirement.

sign_in_not_rightly_placed

## Styling
remove the underline styling. You can do this with CSS to match the original clone.

From this here:
i_forgot_password

To this here:
i_forgot_password_original

PS: Also add the 'I'. You seem to have forgotten about it 😉

File: main.css [hunk 4]
```diff
@@ -0,0 +1,219 @@
+body {
```
Comment (christianotieno) [inline main.css:1]: # [REQUIRED] - GENERAL FEEL
The general feel of your page is not looking exactly the same as in the original page. Have you noticed that?
-
They are slightly different, but the good news is that we can change that.

This is what I am currently seeing.
intuit_clone

I would love to see something close to this:

intuit_original

What is required is general styling.

**Review by christianotieno — CHANGES_REQUESTED** [review]

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @anewman15  & @beyk  :wave:
-
Good job so far! :tada: We have some few changes to do before we proceed (see review comment).
-
Please make carryon with changes and submit another review request.

- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:

- When you have made the specified changes, please submit another code review. Thank you!

*Happy coding :smile:,*
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by christianotieno — COMMENTED** [review]
hello

File: README.md (hunk 1 above)
Comment (anewman15) [reply README.md]: Done

File: mint-signup-form.html (hunk 2 above)
Comment (anewman15) [reply mint-signup-form.html]: Done

File: mint-signup-form.html (hunk 3 above)
Comment (anewman15) [reply mint-signup-form.html:24]: Done

File: main.css (hunk 4 above)
Comment (anewman15) [reply main.css:1]: I changed the width to fixed px from percentage. It should look similar now

**Comment by thsvr** [comment]
### **STATUS: PROJECT APPROVED**  :fireworks: :fireworks: :fireworks: :fireworks: 

Hi @anewman15 and @beyk :smile_cat: :smile_cat: 

Congrats for the hard work, your page looks beautiful, you guys did the required changes asked the the other TSEs and your project was approved :1st_place_medal: :1st_place_medal: :1st_place_medal: 

:v: You guys got the meaning of the names of features branch, BIG congrats for that :100: :1st_place_medal: 

### **SUGGESTION:**

:v: On the readme, would be cool if you guys also add more professional contact info about yourselves, like linkdIn, twitter, etc links and email. 


If you have any questions, don't hesitate to contact me through slack [@thaís Vieira Azevedo](url)

### **Happy coding and good luck on the next project** :computer: 

_Project reviewed by Thaís Vieira (TSE)_

**Comment by anewman15** [comment]
Thanks a lot, @thsvr.
And to @Dandush03 and @christianotieno for your valuable contributions.

My best,
Abdullah

**Pull request merged**